lib/xgu/xgu.h

```diff
@@ -429,17 +429,16 @@ DWORD* xgu_set_vertex_data_array_offset(DWORD* p, XguVertexArray index, const vo
 inline
 DWORD* xgu_elements16(DWORD* p, const uint16_t* elements, unsigned int count) {
     assert(count % 2 == 0);
-    // This one is a bit complicated
     p = push_command(p, NV097_ARRAY_ELEMENTS16, count);
-    memcpy(p, &indices[i], count * sizeof(uint16_t));
+    memcpy(p, &elements[i], count * sizeof(uint16_t));
```

Comment:
`i` still doesn't exist. You should compile your code once in a while.
I think the triangle sample might already work?